spk/rabbitmq/src/rabbitmq.conf

```diff
@@ -0,0 +1 @@
+loopback_users.guest = false
```

Comment:
@DigitalBox98 I have removed `log.dir` definition here, as `LOG_BASE` in `rabbitmq-env.conf` seems to be enough.